gvm/transform/__init__.py

```diff
@@ -35,7 +36,7 @@ def __init__(self):
     def _convert_response(self, response: str) -> etree.Element:
         return etree.XML(response, parser=self._parser)
 
-    def __call__(self, response: str) -> etree.Element:
+    def __call__(self, response: str, gmp=None) -> etree.Element:
```

Comment:
The method signature must be
```suggestion
    def __call__(self, response: str, protocol: GvmProtocol) -> etree.Element:
```
now. The transform could possibly be used with other protocols (like OSP) too.